src/main/java/astra/TaskList.java

```diff
@@ -0,0 +1,111 @@
+package astra;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import astra.task.Task;
+
+/**
+ * Represents a list of tasks.
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public TaskList(Collection<? extends Task> tasks) {
+        this.tasks = new ArrayList<>(tasks);
+    }
+
+    /**
+     * Returns the list of tasks in text file format.
+     *
+     * @return List of tasks in text file format.
+     */
+    public String toText() {
```

Comment:
Perhaps this could be further differentiated from toString() to illustrate that the string is to be converted for the text file format, maybe toFileText()?